aFWall/src/main/java/dev/ukanth/ufirewall/preferences/SecPreferenceFragment.java

```diff
@@ -86,6 +94,20 @@ public void onCreate(Bundle savedInstanceState) {
         setupEnableAdmin(findPreference("enableAdmin"));
 
         //passOption = G.protectionLevel();
+
+        // Hide Fingerprint option if device not support it.
+        if(!FingerprintUtil.isAndroidSupport()) {
```

Comment:
Yes of course it will be better, but no we can't!
Because, [ListPreference API](https://developer.android.com/reference/android/preference/ListPreference.html) does not offer a function for doing that.
Correct me if I'm wrong :)